src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinInetProxyHelper.cs

```diff
@@ -25,16 +27,25 @@ public WinInetProxyHelper()
                     AutoDetect = proxyConfig.AutoDetect;
                     Proxy = Marshal.PtrToStringUni(proxyConfig.Proxy);
                     ProxyBypass = Marshal.PtrToStringUni(proxyConfig.ProxyBypass);
+
+                    WinHttpTraceHelper.Trace(
+                        "WinInetProxyHelper.ctor: AutoConfigUrl={0}, AutoDetect={1}, Proxy={2}, ProxyBypass={3}",
+                        AutoConfigUrl,
+                        AutoDetect,
+                        Proxy,
+                        ProxyBypass);
```

Comment:
Actually, for this call they are: `string`, `bool`, `string`, `string`.  The `IntPtr` are in the proxyConfig struct. But they are converted to .NET strings. I'll add if I can add just specific overloads for these for now.